app/src/main/java/calculator/example/com/kotlincalculator/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,55 @@
+package calculator.example.com.kotlincalculator.mvp.model
```

Comment:
please add proper unit tests to the model class as well

Reply:
Done ¡ 👍